apps/api/src/app/workflows-v2/usecases/generate-preview/generate-preview.usecase.ts

```diff
@@ -190,7 +190,7 @@ export class GeneratePreviewUsecase {
     controlValues: Record<string, unknown>
   ) {
     const payloadSchema = await this.buildPayloadSchema.execute(
```

Comment:
```suggestion
    const variables = await this.extractVariables.execute(
```

not sure about the const value, but we should have the usecase matching the command